.eslintrc.json

```diff
@@ -0,0 +1,54 @@
+{
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "comma-dangle": [2,"never"],
```

Comment:
```js
"max-len": [2, 110, 4]
```
What do you think?

Reply:
Great suggestion, I'm adding this as an alert `"max-len": [1, 110, 4]`